cmake/nlohmann_jsonConfigVersion.cmake.in

```diff
@@ -0,0 +1,16 @@
+set(PACKAGE_VERSION "@PROJECT_VERSION@")
```

Comment:
Could you please explain in a comment why this file is needed (as I understood, to simulate CMake 3.14 behavior)?